package.json

```diff
@@ -40,7 +40,8 @@
     },
     "mocha": {
         "loader": "esmock",
-        "ui": "bdd"
+        "ui": "bdd",
+        "timeout": 10000
```

Comment:
just set it a bigger value - a test was sometimes failing for timeout in my local.